filament/backend/src/vulkan/VulkanDriver.cpp

```diff
@@ -595,6 +595,8 @@ void VulkanDriver::createRenderTargetR(Handle<HwRenderTarget> rth,
             colorTargets[i] = {
```

Comment:
can you add a 
`assert_invariant(color[i].layerCount == layerCount);` here?

Reply:
color[i] is an MRT. depth and stencil are TargetBufferInfo. Neither currently have "layerCount" as a member. Would you want those classes to have a layerCount as well?

Reply:
ah, no I misread it. It's fine without the assert.  Same with the others.